lib/gds_api/exceptions.rb

```diff
@@ -46,6 +46,8 @@ class HTTPConflict < HTTPClientError
   class HTTPUnprocessableEntity < HTTPClientError
   end
 
+  class InvalidUrlException < BaseError; end
```

Comment:
I've got no strong feelings either way, so your vote for `InvalidUrl` is enough to swing the name-o-meter. Updated.